### PR TITLE
swaynag: add details background option

### LINKED ISSUE #5673: Add details background config setting
Add a custom detail background setting instead of relying on border color.

### PR DESCRIPTION
**Pull request by meMuszr**
Adds a new config option for details background for swaynag
Closes #5673

### CODE REVIEW

File: swaynag/config.c [hunk 1]
```diff
@@ -113,20 +115,21 @@ int swaynag_parse_options(int argc, char **argv, struct swaynag *swaynag,
...
-		"  --button-padding padding        Padding for the button text.\n";
+		"  --background RRGGBB[AA]          Background color.\n"
+		"  --border RRGGBB[AA]              Border color.\n"
+		"  --border-bottom RRGGBB[AA]       Bottom border color.\n"
+		"  --button-background RRGGBB[AA]   Button background color.\n"
+		"  --text RRGGBB[AA]                Text color.\n"
+		"  --button-text RRGGBB[AA]         Button text color.\n"
+		"  --border-bottom-size size        Thickness of the bar border.\n"
+		"  --message-padding padding        Padding for the message.\n"
+		"  --details-border-size size		Thickness for the details border.\n"
```
Comment (RedSoxFan) [inline swaynag/config.c:126]: This should remain as spaces between the option and description

File: swaynag/config.c [hunk 2]
```diff
@@ -113,20 +115,21 @@ int swaynag_parse_options(int argc, char **argv, struct swaynag *swaynag,
...
+		"  --details-background RRGGBB[AA]  Details background color.\n"
+		"  --button-border-size size		Thickness for the button border.\n"
```
Comment (RedSoxFan) [inline swaynag/config.c:128]: Same here

**Review by RedSoxFan — COMMENTED** [review]
Aside from the whitespace issues, this LGTM and seems reasonable.

Just as a note since you submitted an issue just to mention, feel free to submit PRs in the future that do not have an issue linked to them. I know they are required in a lot of places, but we don't require them for PR submissions.

File: swaynag/config.c (hunk 1 above)
Comment (TheAvidDev) [reply swaynag/config.c:126]: It also looks like this doesn't need to be changed (the entire section that is). The added option is shorter than the rest.

File: swaynag/swaynag.1.scd [hunk 3]
```diff
@@ -106,6 +106,9 @@ _swaynag_ [options...]
 *--message-padding* <padding>
 	Set the padding for the message.
 
+*--details-background* <RRGGBB[AA]>
+	Set the color for the background for detials.
```
Comment (TheAvidDev) [inline swaynag/swaynag.1.scd:110]:
```suggestion
	Set the color for the background for details.
```

**Review by RedSoxFan — APPROVED** [review]

**Comment by RedSoxFan** [comment]
Thanks!

**Pull request merged**